docs/modules/setup/pages/use-kubernetes.adoc

```diff
@@ -163,3 +165,51 @@ unzip kroki-k8s-singlepod.zip -d kroki-k8s
 You can then create the Kubernetes objects as a batch or individually, similar to what was shown previously.
 
 Validation and uninstall/removal of Kubernetes objects can be followed using the exact same commands as shown previously too.
+
+== Deploy use Helm Charts
+You can alternatively deploy Kroki using Helm Chart provided link:https://github.com/cowboysysop/charts/tree/master/charts/kroki[`here`]. It's much easier to deploy Kroki with charts, since all object definition files are managed by helm, in a style like App. 
```

Comment:
```suggestion
You can alternatively deploy Kroki using https://github.com/cowboysysop/charts/tree/master/charts/kroki[Helm Charts]. It's much easier to deploy Kroki with charts, since all object definition files are managed by Helm. 
```

I don't understand the "in a style like App." ?
Also, could you please declare `https://github.com/cowboysysop/charts/tree/master/charts/kroki` as an attribute (at the top of this file):

```
:uri-helm-charts: https://github.com/cowboysysop/charts/tree/master/charts/kroki
```

And then:

```
You can alternatively deploy Kroki using {uri-helm-charts}[Helm Charts]. 
```

Reply:
For the text `in a style like App.`, I meant to say that install a helm chart is like install an application. Users does not need to dig into the objects/resources inside a chart. 

I removed the text `in a style like App.` here, so that not causing ambiguity.

Also, attribute is added as suggested.